wallet/lib/services/data_stores/remote_data_store.dart

```diff
@@ -1170,6 +1022,15 @@ class RemoteDataStoreImp implements RemoteDataStore {
 
     return result.getOrElse(() => TransactionResponse.initial());
   }
+
+  @override
+  Future<bool> saveUserFeedback({required String walletAddress, required String subject, required String feedback}) async {
+    try {
```

Comment:
Remote data store is not responsible for error catching. Thats what repository do.